### DISCUSSION TITLE
Nvidia Installer broken on 6.0 Odin

### DISCUSSION THREAD

**Discussion by selwynorren** [elementary/appcenter#1544] [Q&A] [answered]
Hi Guys,

Not sure if this is truly bug or something related to my system itself, however on Elementary OS 6.0 20210611 daily image. If I install the nvidia-driver-460 using the App Center, it install fine and well, however the Nvidia X Server settings opens to a blank window with no configuration option inside at all.

After reboot, only my primary monitor switches on and the second monitor switches off and is not detected at all.

Once i remove the driver from App Center and install using CLI with this command:
`sudo ubuntu-drivers install`

This still installs nvidia-driver-460 driver, but now both screens are detected and the Nvidia X Server settings shows up with proper contents.

Essentially the install candidate in App Center does not install everything needed. I have tried this will all the options listed in App Center

**Comment by danirabbit** [comment, marked as answer]
Hey @selwynorren thanks for reporting this. Can you open a new issue report with this information instead of a discussion? Thanks!

**Reply by selwynorren** [reply]
Hey @danrabbit This is where I now feel slightly dumb - lol I am not sure where to open the new issue. This is why I asked it as a discussion first. Would you mind sharing a link with me?

**Reply by zeebok** [reply]
Head to [this link here](https://github.com/elementary/appcenter/issues/new/choose) to get started with a bug report.

**Reply by selwynorren** [reply]
Thx will do so now

**Comment by cassidyjames** [comment]
Just as an update, this has been resolved in AppCenter!
